docs/rules/no-restricted-properties.md

```diff
@@ -0,0 +1,73 @@
+# Disallow certain object properties (no-restricted-properties)
+
+Certain properties on objects may be disallowed in a codebase. This is useful for deprecating an API or restricting usage of a module's methods. This rule looks for accessing a given property key on a given object name, either when reading the property's value or invoking it as a function. You may specify an optional message to indicate an alternative API or a reason for the restriction.
+
+## Rule Details
+
+This rule is aimed at disallowing certain object properties from your code.
```

Comment:
The description and examples gave me initial impression that the rule disallows read access but maybe allows write access to property (for example, left side of assignment statement) but my guess at intended use case and quick look at code suggest it disallows any reference to that member expression?

Also, it is correct and would it be answer a likely question to include computed property form of member expression in examples?